packages/storybook/stories/components/onboarding/FunnelPricing.stories.tsx

```diff
@@ -136,23 +104,23 @@ const commonParams: FunnelStepPricingParameters = {
       'https://media.daily.dev/image/upload/s--QHvr7zBd--/f_auto/v1743491782/public/approved',
     title: '100% money back guarantee',
     content:
-      "We're confident in the quality of our plan. More than a million developers around the world use daily.dev to grow professionally. To get the most out of it, use daily.dev daily. Consume content, explore, and interact with the community. If you still don't love it after 30 days, contact us. We guarantee a full hassle-free refund. No questions asked.",
+      'We\'re confident in the quality of our plan. More than a million developers around the world use daily.dev to grow professionally. To get the most out of it, use daily.dev daily. Consume content, explore, and interact with the community. If you still don\'t love it after 30 days, contact us. We guarantee a full hassle-free refund. No questions asked.',
```

Comment:
Vibe-coding changes?

Reply:
Prettier 👀

Reply:
Ideally we use HTML entities like `&apos`, then we don't need to worry about escaping stuff